cli/replicaset/cconfig.go

```diff
@@ -390,6 +406,47 @@ func (c *CConfigApplication) Demote(ctx DemoteCtx) error {
 	return err
 }
 
+// BootstrapVShard bootstraps vshard for an application by the centralized config orchestrator.
+func (c *CConfigApplication) BootstrapVShard(ctx VShardBootstrapCtx) error {
+	var (
+		lastErr error
+		found   bool
+	)
+	eval := func(instance running.InstanceCtx, evaler connector.Evaler) (bool, error) {
+		roles, err := cconfigGetShardingRoles(evaler)
+		if err != nil {
+			lastErr = fmt.Errorf("failed to get sharding roles: %w", err)
+			// Try again with another instance.
+			return false, nil
+		}
+		isRouter := false
+		for _, role := range roles {
+			if role == "router" {
+				isRouter = true
+				break
+			}
+		}
+		if !isRouter {
+			// Try again with another instance.
+			return false, nil
+		}
+		found = true
+		lastErr = cconfigBootstrapVShard(evaler, ctx.Timeout)
+		return lastErr == nil, nil
+	}
+	err := EvalForeach(c.runningCtx.Instances, InstanceEvalFunc(eval))
+	for _, e := range []error{err, lastErr} {
+		if e != nil {
+			return e
+		}
+	}
+	if !found {
+		return fmt.Errorf("not found any vshard router in replicaset")
+	}
+	log.Info("VShard bootstrapping is done.")
```

Comment:
I believe this message should be printed by the caller of this API.

Reply:
I believe in this too, I put this here because cartridge and cconfig log message are different. Let's discuss it in the comment below.